src/main/java/xyz/thepathfinder/android/ModelRegistry.java

```diff
@@ -13,6 +14,9 @@
 class ModelRegistry {
 
     private static final Logger logger = Logger.getLogger(ModelRegistry.class.getName());
+    static {
+        logger.setLevel(Level.INFO);
```

Comment:
You can access any logger using Logger.getLogger(ClassName.class.getName()); it doesn't matter if it is private. I can still find and change it.